### DISCUSSION TITLE
Uninstall a single release

### DISCUSSION THREAD

**Discussion by dsaintbase1** [helmwave/helmwave#875] [Q&A] [answered]
Hi,

```
helmwave --version
helmwave version 0.36.2
```

I've tried to remove a release from my `vars.yml` file in-order to uninstall it.
After generating the yaml file & launch a build, all that did was to ignore the release all together as if it was never deployed & managed by Helmwave. 

Q: Is there a way to uninstall a release ala `helm uninstall`?

Thanks!

**Comment by zhilyaev** [comment, marked as answer]
Yes, You can use tag

`helmwave down --build --yml -t my-app -p removed_plan`
There are
- `down` will uninstall command
- `--build` generate new plan
- `-p removed_plan` dir to store new plan
- `--yml` will render helmwave.yml.tpl to helmwave.yml

**Reply by dsaintbase1** [reply]
Thanks! It would be great to have this example in the doc (or perhaps it was already there but I've missed it).
